walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -0,0 +1,345 @@
+    title,
+    description,
+    version,
+    id
+  }
+}
+----
+
+[type=verification]
+Does the GraphQL playground allow you run and view results of the query and mutation?
+
+[type=verificationFail]
+----
+Check the logs of the `ionic-showcase-server` pod.
+It should include the string `+connected to messaging service+`.
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+----
+
+
+
+[time=5]
+== Creating a mobile app in Mobile Develope Console
+
+The Mobile Developer Console allows mobile developers to view avaiable mobile services, connect them them to their mobile apps and get the mobile app configurations.
+
+. Log in to the link:{route-mdc-server-host}[Mobile Developer Console, window="_blank"]
+. Click *Create Mobile App* to register your mobile app.
+. Enter the following name for your mobile app:
++
+----
+showcase-app
+----
++
+After the mobile app is registered, it is displayed on the console.
+. Click on the *showcase-app* card to display the configuration screen.
+. Click the *Mobile Services* tab.
+. Click *Bind to App* for *Data Sync* in the *Unbound Services* section.
+. Click Next on Binding screen. 
+. In Parameters Section provide {route-ionic-showcase-server-host} as `Server URL`
+// TODO - we doing this mostly to add datasync
+// Does it even make sense to bind anything else now. We can have them bound in optional steps after?
+. Bind the `Mobile Metrics` and `Identity Management` services to the app by clicking on the "Bind To App" buttons. Use the default binding options, and once the bindings are completed, you will see them in the `Bound Services` section. 
+. Now go back to the `Configuration` tab again, you will see the `mobile-services.json` file is updated with information about the bound services. If you expand the row for each service, you will see the links to each of the service. Feel free to click on the links and explore the dashboard for each of the service.
+// TODO can we use config access instead of manualy getting config. 
+// See https://issues.jboss.org/browse/AEROGEAR-9124  
+// CMD `oc get mobileclient app1 -o json | jq .status` 
+. Copy `mobile-services.json` to some temporary location. We going to need it later
+
+
+[time=10]
+== Run the DataSync Client Showcase PWA application in CodeReady workspace
+
+The AeroGear showcase app demonstrates the capabilities provided by our Services and SDKs
+Starting with Data Sync, Data Sync is a set of Node.js and Client side modules that help you build apps with strong offline and realtime data sync capabilities using GraphQL.
+Showcase application can be run on both local machine and directly inside OpenShift thanks to `Code Ready` integration.
+In this step we going to utilize Code Ready to build mobile application that will connect to backend we run in previous steps.
+
+=== Steps
+
+// TODO how to Access Code Ready?
```

Comment:
I see code ready available on the cluster so we can have it as Walkthorugh resource. 
Is that the right approach?